client/client.go

```diff
@@ -251,12 +260,17 @@ func (c *Client) DeactivateUser(userId string, status string) error {
 	authtoken := "Bearer "+c.authToken
 	req.Header.Add("content-type", "application/json")
 	req.Header.Add("authorization", authtoken)
-	_, err = c.httpClient.Do(req)
+	resp, err := c.httpClient.Do(req)
 	if err != nil {
 		log.Println("[DEACTIVATE/ACTIVATE ERROR]: ",err)
 		return nil
 	}
-	return nil
+	if resp.StatusCode >= 200 && resp.StatusCode <= 299 {
+		return nil
+    	} else {
+		log.Println("[DEACTIVATE/ACTIVATE ERROR]: ",Errors[resp.StatusCode])
```

Comment:
Here the Errors array is used to return error. This will again lead to same incomprehensible error. So, Return error message from the response body.